packages/twitch-helpers/src/lib/twitch.ts

```diff
@@ -151,9 +149,10 @@ export async function fetchUserFollowage(followerId: string, streamerId: string)
  * }
  * ```
  */
-export function checkSignature(algorithm: string, signature: string, data: any) {
-	const hash = createHmac(algorithm, EventSubSecret).update(data).digest('hex');
+export function checkSignature(algorithm: string, signature: string, data: BinaryLike) {
+	if (EventSubSecret === null) throw new Error('Environment variable TWITCH_EVENTSUB_SECRET was not set');
```

Comment:
Extract this to a function `checkEventSubSecret` and call it in all 3 places

Reply:
We can't assert scope variables, so that's not a possibility to remove the null check.